static/src/js/zustand/types.ts

```diff
@@ -264,6 +264,33 @@ export type MapView = {
   zoom: number
 }
 
+export type MapLayer = {
+  /** The product identifier for the map layer */
+  product: string
+  /** The display title for the map layer */
+  title?: string
+  /** The image format of the layer (e.g., 'image/png', 'image/jpeg') */
+  format?: string
+  /** The temporal period of the layer (e.g., 'daily', 'monthly') */
+  layerPeriod?: string
+  /** Resolution for Antarctic projection */
+  antarctic_resolution?: string
```

Comment:
Should we make sure these properties are kebab-cased before making their way to the store? @macrouch @eudoroolivares2016 thoughts?

Reply:
Uh not sure I think this will change once we integrate umm-viz